extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -171,7 +172,11 @@ export class AmpA4A extends AMP.BaseElement {
     // from cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
     /** @private {?ArrayBuffer} */
     this.creativeBody_ = null;
-    /** @private {?string} */
+    /**
+     * Note(keithwrightbos) - ensure the default here is null so that ios
+     * uses safeframe when response header is not specified.
+     * @private {?string}
+     */
```

Comment:
Done.  I was saving the isIos regexp execution if we got back the response header but I suppose we should assume going forward that would be a rarity.  I also had to duplicate this in unlayoutcallback to ensure we reset properly.  Let me know if you still think this is worth doing.